src/js/disability-benefits/containers/StatusPage.jsx

```diff
@@ -7,15 +7,15 @@ import AskVAToDecide from '../components/AskVAToDecide';
 import ClaimsTimeline from '../components/ClaimsTimeline';
 import ClaimDetailLayout from '../components/ClaimDetailLayout';
 import { setUpPage, isTab, scrollToTop, setFocus } from '../utils/page';
-import { itemsNeedingAttentionFromVet, getCompletedDate } from '../utils/helpers';
+import { itemsNeedingAttentionFromVet, getClaimType, getCompletedDate } from '../utils/helpers';
 
 import { clearNotification } from '../actions';
 
 const FIRST_GATHERING_EVIDENCE_PHASE = 3;
 
 class StatusPage extends React.Component {
   componentDidMount() {
-    document.title = 'Status - Your Disability Compensation Claim';
+    document.title = `Status - Your ${getClaimType(this.props.claim)} Claim`;
```

Comment:
Yeah, this is somewhat awkward. You'll need to check `this.props.loading` in `componentDidMount()` and use a generic title if it's still loading. That will catch the case where someone has navigated directly to this page and data is still loading or if they navigated here from a page that already loaded the claim. Then you will need to use `componentDidUpdate()` to detect when loading finishes by comparing `this.props.loading` and `prevProps.loading`. You can see an example in the current code for that method in this component (though with an extra check you won't need for the title).